pages/events/[id].js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import Layout from '../../components/Layouts/layout';
+import { api } from '../../lib/api';
+import {
+  Box,
+  Center,
+  Flex,
+  Heading,
+  IconButton,
+  Text,
+  useMediaQuery,
+} from '@chakra-ui/react';
+import { LargeEntranceAnimation } from '../../components/Views/Events/singleEvent/LargeEntranceAnimation';
+import { SmallEntranceAnimation } from '../../components/Views/Events/singleEvent/SmallEntranceAnimation';
+import { Banner } from '../../components/Views/Events/singleEvent/Banner';
+import { CurrencyIcon, SaveIcon } from '../../assets';
+import { MainSection } from '../../components/Views/Events/singleEvent/MainSection';
+import { Ticket } from '../../components/Views/Events/singleEvent/Ticket';
+
+export async function getStaticPaths() {
+  if (process.env.NEXT_PUBLIC_SKIP_BUILD_STATIC_GENERATION) {
+    return {
+      paths: [],
+      fallback: 'blocking',
+    };
+  }
+  const res = await fetch(`${api}/events`);
+  const events = await res.json();
+
+  const paths = events.map(event => ({
+    params: { id: event._id },
```

Comment:
Would be a good idea to support also the event.longUrl and event.shortUrl, in the backend call to get all events this two properties can be used to create the static paths. This the longUrl is composed by the event name separated by dashes and the shortUrl. If the static paths are created this way the full url should look like http://localhost:3000/event/corona-capital-2022$$TYmcj9tLBP in the backend you can use the shortId to query for specific event information

Reply:
Got it, I'll work on that at the same time that I'm working on the Form to create new events.